src/changelog/unreleased.md

```diff
@@ -132,6 +132,7 @@ changelog entry.
   - `Window::set_max_inner_size` to `set_max_surface_size`.
 
   To migrate, you can probably just replace all instances of `inner_size` with `surface_size` in your codebase.
+- On X11, set an "area" attribute on XIM input connection to convey the cursor area.
```

Comment:
It's in `Added`.

Reply:
Okay, I'll move it to Added. It seemed like it could go either way to me (since the previous change to XNSpotLocation was in Changed.